models/doubly_linked_list.go

```diff
@@ -0,0 +1,56 @@
+package models
+
+import "fmt"
+
+// Node represents a single node in the doubly linked list
+type Node struct {
+	data int   // The data stored in the node, can be of any data type
+	next *Node // A pointer to the next node in the list
+	prev *Node // A pointer to the previous node in the list
+}
+
+// DoublyLinkedList represents a doubly linked list
+type DoublyLinkedList struct {
+	head *Node // A pointer to the first node (head) of the linked list.
+}
+
+// AddNode adds a new node to the end of the linked list
+func (list *DoublyLinkedList) AddNodes(data int) {
```

Comment:
You need to change the signature of this method to : 

AddNotes(data []int)

Reply:
requested changes addressed.